src/data/list/range.lean

```diff
@@ -8,13 +8,20 @@ import data.list.nodup
 import data.list.of_fn
 import data.list.zip
 
-open nat
+/-!
+# Ranges of naturals
```

Comment:
```suggestion
# Ranges of naturals as lists
```

Perhaps the docstring should mention that the definitions mentioned are all in `data.list.defs`?

Actually, it looks like `list.fin_range` is a definition which exists in this file (and it's a range of elements of `fin n`, not `nat`, though I'm not sure that's worth changing the title of this file.) However, maybe it should also be mentioned in the module doc string.